pkg/docker/cockpit-atomic-storage

```diff
@@ -443,7 +452,10 @@ def get_dss_vgroup():
         for l in open("/proc/mounts", "r").readlines():
             fields = l.split()
             if fields[1] == "/" and fields[0].startswith("/dev"):
-                vgroup = check_output([ "lvs", "--noheadings", "-o",  "vg_name", fields[0]]).strip()
+                try:
+                    vgroup = check_output([ "lvs", "--noheadings", "-o",  "vg_name", fields[0]]).strip()
+                except:
```

Comment:
I would suggest catching the specific type of error that can happen here. Whether it's an IOError or a CalledProcessError